arrocke/gloss-translation
# feat: approve glosses

## What has changed

* Glosses now have a state with values `UNAPPROVED` and `APPROVED`
* Enter approves the gloss, escape unapproved
* Gloss suggestion dropdown is only previously approved glosses



## Connected Issues



closes #203 

## QA Steps



1. Confirm that enter approves the gloss
2. Confirm that tab does not approve
3. Confirm that selecting suggestion with mouse approves gloss
4. Confirm that escape unapproves gloss
5. Confirm that suggestion list is just previously approved glosses

## Post-Deployment
